src/tree.scss

```diff
@@ -14,46 +12,32 @@ $block: #{$fd-namespace}-tree;
 
 .#{$block} {
   // LOCAL VARS (set all themeable properties, always include !default)
-  $fd-tree-background-color: fd-color("background", 2) !default;
-  $fd-tree-border-color: fd-color("neutral", 2) !default;
+  $fd-tree-background-color: #fff !default;
+  $fd-tree-border-color: #eeeeef !default;
   $fd-tree-border-width: 0 !default;
-  $fd-tree-link-color: fd-color(action, 1) !default;
-  $fd-tree-cell-spacing: $fd-width--gutter !default;
+  $fd-tree-link-color: #0a6ed1 !default;
 
-  --fd-tree-cell-spacing: var(--fd-width-gutter);
-  $fd-tree-cell-padding: fd-space(3) !default;
-  $fd-tree-header-color: fd-color("text", 3) !default;
-  $fd-tree-header-background-color: fd-color("neutral", 2) !default;
-  $fd-tree-row-background-color--hover: fd-color-state("hover") !default;
+  $fd-tree-header-color: #6a6d70 !default;
+  $fd-tree-header-background-color: #eeeeef !default;
+  $fd-tree-row-background-color--hover: #fafafa !default;
 
   $fd-tree-control-width: 18px !default;
   $fd-tree-control-button-width-height: 34px !default;
```

Comment:
in rem?

Reply:
I left it as is as we are refactoring it